src/containers/NFT/NFTTransactions/NFTTransactions.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { useParams, useRouteMatch } from 'react-router'
+import './styles.scss'
+import Tabs from '../../shared/components/Tabs'
+import { getBuyNFToffers, getSellNFToffers } from '../../../rippled/lib/rippled'
+import Offers from './Offers'
+import Transactions from './Transactions'
+
+interface Props {
+  tokenId: string
+}
+
+const NFTTransactions = (props: Props) => {
```

Comment:
This should not be named NFTransactions it contains more than just transactions.  Maybe NFTTabs